src/fsharp/FSharp.Core/map.fsi

```diff
@@ -85,6 +85,9 @@ namespace Microsoft.FSharp.Collections
         /// <returns><c>true</c> if the value is present, <c>false</c> if not.</returns>
         member TryGetValue: key:'Key * [<System.Runtime.InteropServices.Out>] value:byref<'Value> -> bool
 
+        member Keys : 'Key seq 
```

Comment:
Documentation for these please

Reply:
Use `seq<'Key>` in FSharp.Core please

Reply:
I added a xml doc and hunted all the seq types in all the places to switch them to this form